### DISCUSSION TITLE
How can I publish an action to the marketplace? There is no banner as described in the github docs for creating actions

### DISCUSSION THREAD

**Discussion by binomialstew** [community/community#58195] [Actions] [answered]
### Select Topic Area

Question

### Body

I am following the instructions [here](https://docs.github.com/en/actions/creating-actions/publishing-actions-in-github-marketplace#publishing-an-action) to publish an action, but I see no banner as described:

![Screen Shot 2023-06-15 at 6.03.27 PM.png](https://github.zendesk.com/attachments/token/WAX1J6XZHmgz3ulj1I8V4VOct/?name=Screen+Shot+2023-06-15+at+6.03.27+PM.png)

The action I would like to publish:
https://github.com/binomialstew/conventional-changelog-action/blob/feature/infile-addition/action.yml

I have read through the other discussions that seem similar to this issue, but they only mention a need to sign up for github actions beta, which I don't think is still a requirement. I can't find any type of sign up or flag in my settings that seems related to this.

I've tried using both `actions.yml` and `actions.yaml`:
https://github.com/orgs/community/discussions/24965

I have created releases both from UI and also from a PR merge. They do not show the "marketplace" tag. There is an actions workflow that runs a release job with the `actions/create-release@v1` action that seemed to run successfully (ran without error and created a release).
<img width="1262" alt="Screen Shot 2023-06-15 at 6 34 09 PM" src="https://github.com/community/community/assets/6164653/f7545f47-3604-4631-a49b-9b221c905ad1">

How can I publish this action?

**Comment by binomialstew** [comment, marked as answer]
I had been checking this page consistently for 8 hours yesterday after making my last change to the `actions.yml` and did not see the banner show up. Strangely, the banner showed up this morning with no changes to the repo. The only thing that had changed was I got a response back from support, but they said they had made no changes.

They said it might have been a one-time lag on their end.

**Comment by Armadillidiid** [comment]
I'm experiencing this same issue sadly. It's been over 3 days now, yet, I can't see the banner.
